recipes/veloxchem/meta.yaml

```diff
@@ -0,0 +1,107 @@
+{% set name = "veloxchem" %}
+{% set version = "1.0" %}
+{% set pre = "rc3" %}
+{% set sha256 = "ed22aeb72d832f0daccffa393a1a349dab65fc85cefbb6a069933d3b51ad7ffd" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://gitlab.com/veloxchem/veloxchem/-/archive/v{{ version }}-{{ pre }}/veloxchem-v{{ version }}-{{ pre }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - patches/geometric-as-posix.patch
+
+build:
+  number: 0
+  script_env:
+    - OMPI_MCA_plm_rsh_agent=  # [mpi == 'openmpi' and linux]
+  binary_relocation: true
+  entry_points:
+    - vlx=veloxchem.main:main
+  run_exports:
+    - {{ pin_subpackage('veloxchem', max_pin='x.x') }}
```

Comment:
```suggestion
```
Run exports are only needed if you are shipping headers and shared libraries (located in $PREFIX/lib), but you haven't existence tested those things in the test section. Therefore, nobody should be linking against your package and the run_export is moot.